src/kirby/components/list/list.component.html

```diff
@@ -14,38 +16,42 @@
       </ion-item-divider>
 
       <div class="list-items">
-        <ion-item *ngFor="let item of section.items; let idx = index;trackBy: trackByFn"
-                  [class.has-divider]="showDivider"
-                  [class.selectable]="isSelectable"
-                  [class.selected]="markSelectedRow && selectedItem && item._id === selectedItem._id"
-                  [attr.role]="isSelectable ? 'button' : null"
-                  [class.last-in-section]="idx === section.items.length - 1"
-                  (click)="onItemSelect(item)"
-                  tabindex="0"
-                  [kirbyListItemColor]="getItemColor"
-                  [item]="item"
-                  keyHandler>
-          <ng-container *ngTemplateOutlet="listItemTemplate; context: { $implicit: item }"></ng-container>
-          <ng-container *ngTemplateOutlet="listFlexItemTemplate; context: { $implicit: item }"></ng-container>
-        </ion-item>
+        <ion-item-sliding *ngFor="let item of section.items; let idx = index;trackBy: trackByFn" [disabled]="isSwipingDisabled">
```

Comment:
Should we consider performance regarding always wrapping list items in an item-sliding wrapper?
Maybe moving the content of this tag to a template and conditionally wrapping in `ion-item-sliding`:
(We could do this later as part of a general performance overhaul on the list...)
Ohhh - I see we are already duplicating the `ion-item` 😱  I think NOW would be a good time to DRY that up....